web/themes/custom/novel/templates/components/list-teaser-stacked.html.twig

```diff
@@ -0,0 +1,12 @@
+<a href="{{ url }}" class="content-list-item-stacked arrow__hover--right-small">
```

Comment:
This uses a different approach to adding attributes compared to `list-teaser`.

I wonder if it is intentional.